### PR TITLE
Create page for Online-Zoo

### PR DESCRIPTION
**Pull request by VolhaVaitulevich**

### CODE REVIEW

File: pages/styles.css [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+  text-align: center;
+  font-size: 19px;
+  line-height: 120%;
+  background-image: url("logo_image.svg");
+  background-repeat: no-repeat;
+  background-position: bottom 20% left 8%;
+}
+
+.navigation-tabs__link {
+  float: left;
```
Comment (siarhei-papkou) [inline pages/styles.css:43]: Try to do it using `flex`

File: pages/styles.css [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+.page-content__how-it-works-img {
+  width: 40%;
+}
+
+.page-content__flower1 {
+  width: 15%;
+}
+
+h1 {
+  font-family: Roboto;
```
Comment (siarhei-papkou) [inline pages/styles.css:159]: You cand define font for whole page in `*` section

File: pages/styles.css [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+  color: white;
+  background-color: #f9804b;
+  font-family: Roboto;
+  font-size: 16px;
+  border: 0;
+  border-radius: 5px;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  padding-left: 45px;
+  padding-right: 45px;
```
Comment (siarhei-papkou) [inline pages/styles.css:263]: you can use shorter syntax `padding: 15px 45px`

File: pages/styles.css [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.footer__navigation-tabs {
+  align-self: flex-end;
+}
+
+.footer__navigation-tabs__link {
+  float: left;
```
Comment (siarhei-papkou) [inline pages/styles.css:330]: Let's avoid the `float` usage

File: pages/styles.css [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+}
+
+.footer__contacts-circle-instagram {
+  background-image: url("logo_instagram.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.footer__contacts-circle-youtube {
+  background-image: url("logo_youtube.svg");
```
Comment (siarhei-papkou) [inline pages/styles.css:320]: It will be nice to place images in dedicated folder

Comment (VolhaVaitulevich) [reply pages/styles.css]: Fixed in a new commit, created a new folder 'images', and moved all the images there.

File: pages/styles.css (hunk 4 above)
Comment (VolhaVaitulevich) [reply pages/styles.css]: Fixed in a new commit, updated with 'flex'

File: pages/styles.css (hunk 3 above)
Comment (VolhaVaitulevich) [reply pages/styles.css]: Updated in a new commit

File: pages/styles.css (hunk 1 above)
Comment (VolhaVaitulevich) [reply pages/styles.css]: Fixed in a new commit.

**Review by siarhei-papkou — APPROVED** [review]

**Pull request merged**